README.md

```diff
@@ -7,14 +7,43 @@ Details on each command can be found in the [CLI User Manual](https://keychain.o
 
 ## Quick start
 
-requires: [docker](https://www.docker.com/)
+Recommended system requirements: 
+- GNU/Linux OS with [docker](https://www.docker.com/) for containerized operation
+- node 18.19.1 and npm 10.8.2 or newer for manual and local operation
```

Comment:
We build kc containers with node 18.15.0, it doesn't make sense to require a higher version

The services may run with node 19+ but the unit tests won't run due to some jest incompatibility